README.md

```diff
@@ -229,6 +229,10 @@ Every time you pull new changes down, kill docker (if it's running) and re-run:
 HostedGPT requires these services to be running:
 
 - Postgres ([installation instructions](https://www.postgresql.org/download/))
+  - Use environment variable `DATABASE_URL` to specify the connection parameters
+  - *Make sure to set that environment variable in your setup!*
+  - Recommended (development): `DATABASE_URL=postgres://app:secret@localhost/hostedgpt_development`
+  - Recommended (test): `DATABASE_URL=postgres://app:secret@localhost/hostedgpt_test`
```

Comment:
```suggestion
- Postgres (`brew install imagemagick` or other [install instructions](https://www.postgresql.org/download/))
  - By default postgres will create a default user and things should just work, but if you configure a specific username and password then set the environment `DATABASE_URL=postgres://username:password@localhost/hostedgpt_development` (replacing username & password with your values, optionally you can change the database name)
```

Reply:
@krschacht I changed database.yml to be more explicit that we're using DATABASE_URL, per the recommendation in the Rails Guide [3.21 Connection Preference](https://edgeguides.rubyonrails.org/configuring.html#connection-preference)

(So yes - you are correct that DATABASE_URL works as-is; editing database.yml is just to reduce developer confusion).

New commit added!

Reply:
@draknor Got it. Yes, you’re right, I think it’s good that we explicitly set `url:` as you’ve done. That makes it easier to discover. In fact, let’s even add a comment that gives an example of the format for the URL.

The thing I worry about is if any users have made use of the ENV which are being removed from database.yml, we might break their setup with this release. Reading the Rails Guide that you linked to, it sounds like details from DATABASE_URL will take precedence over any explicitly specified things. I’m just inclined to add back all the red removals from database.yml (the default `host` and the default `database` and what not).